### ISSUE TITLE
What is this running?

### ISSUE THREAD

**Issue by waldoj** [codeforamerica/PoliceOpenDataCensus#2]
It sure looks like y'all have rewritten Open Knowledge's Open Data Census. Is that right? I'd love to look at using this for an open data census, for which dealing with Open Knowledge's system has proven frustrating. Is this a deployment of a content-neutral system, or is this census the project itself? If the latter, is there any reason why I couldn't use this for my own, non-police, open data census?

**Comment by creade** [comment]
Hey!

The Police Open Data Census is the project itself, but there's not a whole lot of surface area that would need to be changed to adopt it to some other topic. We borrowed a lot of (chiefly design-related) ideas from a version back of the Open Knowledge's Open Data Census but we also dropped their backend in favor of GitHub hosting, Google Docs for storing data and tabletop.js. If you don't mind a lot of manual updating of the backing spreadsheets (and the occasional Google Drive outage) that model has worked well for us. 

Feel free to reach out if you've got any questions!
